experimental/CliffordAlgOrd/src/IO.jl

```diff
@@ -0,0 +1,73 @@
+##################################################
+#
+#  Clifford algebra and elements
+#
+##################################################
+ 
+##### Algebra #####
+function Base.show(io::IO, ::MIME"text/plain", C::CliffordAlgebra)
+  io = pretty(io)
+  print(io, "Clifford algebra of quadratic space with Gram matrix\n")
+  print(io, Indent())
+  show(io, "text/plain", gram_matrix(C))
+  print(io, Dedent(), "\ndefined over $(base_ring(C))")
+end
+ 
+function Base.show(io::IO, C::CliffordAlgebra)
+  if is_terse(io)
+    print(io,"Clifford algebra")
+  else
+    print(terse(io), "Clifford algebra over ")
+    print(terse(io), base_ring(C))
+  end
+end
+
+##### Elements #####
+function Base.show(io::IO, x::CliffordAlgebraElem)
+  print(io, "[")
+  foreach(y -> print(io, "$y "), coefficients(x)[1:(end - 1)])
+  print(io, "$(coefficients(x)[end])]")
```

Comment:
```suggestion
  print(io, "[")
  join(io, coefficients(x), " ")
  print(io, "]")
```
I think you wanna do this? Seems to be more readable to me

Reply:
Or `b = coefficients(x); print(IOContext(io, :typeinfo=>typeof(b)), b)`